### ISSUE TITLE
showFileHeader generates blank lines

### ISSUE THREAD

**Issue by in-in** [amzn/style-dictionary#418]
`"showFileHeader": false` generates blank lines in sass files

### Steps to reproduce

```shell
npx style-dictionary init basic
```

edit `config.json`
```diff
{
  "destination": "_variables.scss",
- "format": "scss/variables",
+ "format": "scss/map-flat",
+ "options": {"showFileHeader": false}
}
```
run
```shell
npx style-dictionary build
```
open `build/scss/_variables.scss`

### Expected behaviour
There shouldn't be any blank lines

### Actual behaviour
There are two blank lines.
![201001586429952](https://user-images.githubusercontent.com/8797432/78889551-884c9b80-7a6c-11ea-9996-c7fc852f3f83.png)

**Comment by dbanksdesign** [comment]
Ah I see, yea it looks like we could move this line up into the conditional that shows the file header:
https://github.com/amzn/style-dictionary/blob/master/lib/common/templates/scss/map-flat.template#L28

**Comment by chazzmoney** [comment]
I think we could change this; however - is this an actual incorrect format?

I want to make sure we understand your interest/need to remove the blank lines correctly before we do a code change.

**Comment by chazzmoney** [comment]
Fixed in 3.0
